src/main/java/uk/gov/hmcts/darts/common/entity/base/CreatedModifiedBaseEntity.java

```diff
@@ -21,7 +20,7 @@ public class CreatedModifiedBaseEntity extends CreatedBaseEntity {
     @Column(name = "last_modified_ts")
     private OffsetDateTime lastModifiedDateTime;
 
-    @ManyToOne(fetch = FetchType.LAZY)
+    @ManyToOne
```

Comment:
was there a reason the Lazy loading was removed?

Reply:
yep I've mentioned in the PR description. Happy to have a chat about it if you have concerns Chris

Reply:
i think we should be leaving the Lazy loading. this class is used by practically every entity and so will result in extra db calls all over the system for no reason, as nothing needs to know the users name for functionality.
I don't think we should be changing code behaviour just to help make a test a little shorter.

Reply:
fair enough, I've reverted the change.